test/staking/StakeManager.test.js

```diff
@@ -784,6 +785,119 @@ contract('StakeManager: Heimdall fee', async function (accounts) {
         wallets[2].getAddressString()
       ), fee)
     })
+
+    it('More than 2/3rd vote for yes and the rest vote no', async function () {
+      wallets = generateFirstWallets(mnemonics, 4)
+
+      const validators = [1, 2, 3, 4]
+      const amount = web3.utils.toWei('4')
+      let fee = web3.utils.toWei('2')
+      let stakeAmount = web3.utils.toWei('2')
+      let counter = 0
+
+      wallets.forEach( async (wallet) => {
+        await stakeToken.mint( wallet.getAddressString(), amount)
+        await stakeToken.approve(stakeManager.address, amount, {
+          from: wallet.getAddressString()
+        })
+        await stakeManager.stake(stakeAmount, fee, false, wallet.getPublicKeyString(), {
+          from: wallet.getAddressString()
+        })
+        counter += 1
+      })
+
+      if (counter === 4) {
```

Comment:
Not sure why `counter` variable is needed!

Reply:
to make the code synchronous

Reply:
done